motor.cpp

```diff
@@ -23,7 +23,15 @@ void Motor::stop(int motor, int motor2, int motor3, int motor4) {
     }
 }
 
+void Motor::go_all(int speeds[], int directions[]) {
+    for (int i = 1; i <= MOTOR_NUM ; i++)
+        go(i, directions[i], ((speeds[i] < 0)?0:speeds[i]));
```

Comment:
Once again, we don't need to do the check here, since the check is done at `Motor::go`, like 4 lines below this one...